lib/IRGen/IRGenModule.cpp

```diff
@@ -610,6 +610,10 @@ llvm::Constant *swift::getWrapperFn(llvm::Module &Module,
                                                   RETURNS, ARGS, ATTRS)        \
   FUNCTION_IMPL(ID, NAME, CC, QUOTE(RETURNS), QUOTE(ARGS), QUOTE(ATTRS))
 
+#define FUNCTION_WITH_GLOBAL_SYMBOL_FOR_CONV_SwiftCC(ID, NAME, SYMBOL, CC,        \
+                                                  RETURNS, ARGS, ATTRS)        \
+  FUNCTION_IMPL(ID, NAME, CC, QUOTE(RETURNS), QUOTE(ARGS), QUOTE(ATTRS))
```

Comment:
Isn't it a problem to drop the SYMBOL part here? I thought that's what you were trying to fix on Windows.

Reply:
It's not actually unused. Like for the other calling conventions in this file, the `SYMBOL` arg is dropped here but used in [RuntimeEntrySymbols.cpp](stdlib/public/runtime/RuntimeEntrySymbols.cpp). I only had to add this macro here because SwiftCC wasn't previously being used on a global symbol.

This 'fix' for Windows is really a unification across all platforms by making sure the calling convention for these functions is guaranteed to be compatible with Swift.

Reply:
Oops, didn't think to check for that. Thanks for the explanation.